### DISCUSSION TITLE
SSL with quasar build command

### DISCUSSION THREAD

**Discussion by operez2000** [quasarframework/quasar#17488] [CLI - SSR mode] [answered]
Hi, I've following the https://quasar.dev/quasar-cli-webpack/quasar-config-file/ documentation to work with SSL and it works perfect with **_quasar dev -m ssr_**, when I go to **_quasar build -m ssr_** it does not work, how can I make it work with build?
Thanks

**Comment by yusufkandemir** [comment, marked as answer]
As you've found out, https://github.com/quasarframework/quasar/discussions/13287#discussioncomment-2659874
